[2025-02-05 15:28:40,734][lineage_manta_implement.py:139][ERROR] Error in deploy_lineage_manta: 'ValueObjectType'
[2025-02-05 15:30:27,759][lineage_manta_implement.py:139][ERROR] Error in deploy_lineage_manta: 'ValueObjectType'
